gabriel-portfolio/portfolio/src/main/webapp/photo-album.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gabriel's Photo Album</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body id="photo-album">
+    <div id="content">
+
+      <div class="topnav">
+        <a href="/index.html">Home</a>
+        <a class = "active" href="/photo-album.html">Photo Album</a>
+      </div>
+
+      <h1 class="h1-photo-album">WELCOME TO MY PHOTO ALBUM!</h1>
+
+      <div id="birthday-pic">
+        <p> 
+          This is me, my high school CS teacher, and my best friend at his 18th birthday party.
+        </p>
+        <img src="/photos/birthday-pic.png" alt="This is me" width="300" height="auto">
```

Comment:
width="300" and height="auto" are used multiple times. Can you move it to a css file and reuse it?

Reply:
Resolved